### ISSUE TITLE
Isn't there a limit to the length if you send View Component's model through Header as a Post Method?

### ISSUE THREAD

**Issue by won5572** [hydrostack/hydro#15]
Isn't there a limit to the length if you send View Component's model through Requet Header as a Post Method?
Is there a reason why you didn't implement it as a Request Body?

I've solved the problem with Transient attributes, 
but I think there will be a lot of data from the model. Is there a good solution in this case?

[additional]
I'm testing hydro-sale's toast using Hydro-Trigger. However, it breaks when I type in Korean as Message. The HttpHeader protocol says that only Ansi code is available.
When I tried to solve this problem, I had to encode and decode it with base64.
Wouldn't it be possible to solve this at once with a request-body or response-body without using a header.

```csharp
  private void PopulateDispatchers()
  {
      if (!_dispatchEvents.Any())
      {
          return;
      }

      var data = _dispatchEvents
          .Select(e => new { name = e.Name, data = e.Data, scope = e.Scope, operationId = e.OperationId })
          .ToList();

var json = JsonConvert.SerializeObject(data);
var bytes = Encoding.UTF8.GetBytes(json);
var base64Json = Convert.ToBase64String(bytes);

HttpContext.Response.Headers.TryAdd(HydroConsts.ResponseHeaders.Trigger, base64Json);
  }
```
```javascript
const triggerHeader = response.headers.get('Hydro-Trigger');
if (triggerHeader) {

    // Base64 to ArrayBuffer
    const binaryString = window.atob(triggerHeader);
    const len = binaryString.length;
    const bytes = new Uint8Array(len);
    for (let i = 0; i < len; i++) {
        bytes[i] = binaryString.charCodeAt(i);
    }

    // decoding
    const decoder = new TextDecoder('utf-8');
    const decodedHeader = decoder.decode(bytes);
    const triggers = JSON.parse(decodedHeader);

    triggers.forEach(trigger => {
        if (trigger.scope === 'parent' && !parentComponent) {
            return;
        }

        const eventScope = trigger.scope === 'parent' ? parentComponent.id : 'global';
        const eventName = `${eventScope}:${trigger.name}`;
        const eventData = {
            detail: {
                data: trigger.data,
                operationId: trigger.operationId
            }
        };

        document.dispatchEvent(new CustomEvent(eventName, eventData));
    });
}
```

**Comment by kjeske** [comment]
That's a good point, thanks. I'm already working on moving the data from the headers to the body. Will update the issue once it's ready.

**Comment by kjeske** [comment]
Fixed in v0.12.0
